CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+### 4.0.0
+
+* `relativePaths` is now automatically set to `false` when `publicPath` is used
+* `publicPath` now uses `webpack.config.js`'s `output.publicPath` by default (when not set in `OfflinePlugin` itself)
+* Added `AppCache.output` which is replacement for `AppCache.directory`. The last is deprecated now
+* Added `ServiceWorker.publicLocation` and `AppCache.publicLocation`
```

Comment:
@MoOx would be it clearer if it were called `publicURL` or `publicPath`? This option means to provide the URL to find SW/AppCache to register, e.g. if you put it in custom location.